.github/workflows/main.yml

```diff
@@ -310,9 +310,25 @@ jobs:
         strip -g target/x86_64-unknown-linux-musl/release/wasm2es6js
     - uses: actions/upload-artifact@v2
       with:
-        name: dist_linux
+        name: dist_linux_x86_64
         path: "target/x86_64-unknown-linux-musl/release/wasm*"
 
+  dist_linux_aarch64:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup update --no-self-update stable && rustup default stable
+      - run: rustup target add aarch64-unknown-linux-musl
+      - run: sudo apt update -y && sudo apt install musl-tools -y
+      - run: sudo snap install --beta --classic zig
+      - run: cargo install cargo-zigbuild
```

Comment:
Could this build be done without these tools? I don't know what these are and I think it'd be best ideally to use the distro-provided tools if possible.

Reply:
Do you mean cargo-zigbuild or both zig and [cargo-zigbuild](https://github.com/messense/cargo-zigbuild)?

I could try using distro-provided tools but using zig+cargo-zigbuil is just very [convenient](https://actually.fyi/posts/zig-makes-rust-cross-compilation-just-work/) for cross compilation

Reply:
Ideally I think it'd be nice to avoid both, is there no built-in way to produce aarch64 binaries on github actions or on ubuntu?

Reply:
I am able to compile to aarch64-unknown-linux-gnu using the distro-provided gcc-aarch64-linux-gnu.

For musl I haven't had success. Any suggestion? 

So far, this is the error log of my failed attempt:

```
error: failed to run custom build command for `libz-sys v1.1.8`

Caused by:
  process didn't exit successfully: `/home/runner/work/wasm-bindgen/wasm-bindgen/target/release/build/libz-sys-5d870231b03c8667/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=LIBZ_SYS_STATIC
  cargo:rerun-if-changed=build.rs
  cargo:rerun-if-env-changed=ZLIB_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_ALLOW_CROSS_aarch64-unknown-linux-musl
  cargo:rerun-if-env-changed=PKG_CONFIG_ALLOW_CROSS_aarch64_unknown_linux_musl
  cargo:rerun-if-env-changed=TARGET_PKG_CONFIG_ALLOW_CROSS
  cargo:rerun-if-env-changed=PKG_CONFIG_ALLOW_CROSS
  cargo:rerun-if-env-changed=PKG_CONFIG_aarch64-unknown-linux-musl
  cargo:rerun-if-env-changed=PKG_CONFIG_aarch64_unknown_linux_musl
  cargo:rerun-if-env-changed=TARGET_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_aarch64-unknown-linux-musl
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_aarch64_unknown_linux_musl
  cargo:rerun-if-env-changed=TARGET_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo-warning=pkg-config has not been configured to support cross-compilation.

  Install a sysroot for the target platform and configure it via
  PKG_CONFIG_SYSROOT_DIR and PKG_CONFIG_PATH, or install a
  cross-compiling wrapper for pkg-config and set it via
  PKG_CONFIG environment variable.
  TARGET = Some("aarch64-unknown-linux-musl")
  OPT_LEVEL = Some("3")
  HOST = Some("x86_64-unknown-linux-gnu")
  CC_aarch64-unknown-linux-musl = None
  CC_aarch64_unknown_linux_musl = None
  TARGET_CC = None
  CC = None
  CROSS_COMPILE = None
  CFLAGS_aarch64-unknown-linux-musl = None
  CFLAGS_aarch64_unknown_linux_musl = None
  TARGET_CFLAGS = None
  CFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("false")
  CARGO_CFG_TARGET_FEATURE = Some("neon,pmuv3")
  running: "aarch64-linux-musl-gcc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-I" "src/zlib" "-fvisibility=hidden" "-DSTDC" "-D_LARGEFILE64_SOURCE" "-D_POSIX_SOURCE" "-o" "/home/runner/work/wasm-bindgen/wasm-bindgen/target/aarch64-unknown-linux-musl/release/build/libz-sys-d7ccf3b9d46c7d7a/out/lib/src/zlib/adler32.o" "-c" "src/zlib/adler32.c"

  --- stderr


  error occurred: Failed to find tool. Is `aarch64-linux-musl-gcc` installed?
```